### ISSUE TITLE
Meteor 2.5.1 compatibility

### ISSUE THREAD

**Issue by bratelefant** [activitree/meteor-push#47]
Hi there,

tried to update Meteor to 2.5.1; I also use cordova-plugin-androidx-adapter@1.1.3. Bulding with meteor-push@2.0.10 yields several error (s. below).

When I manually change the dependency in the gradle file under "cordova-support-google-services" for google services to `classpath 'com.google.gms:google-services:4.3.10'´ building seems to work fine, and at least push tokens get inserted into mongo db. (did no further testing atm).

Is there a chance that 1.) this is a valid solution and 2.) this gets merged?

**Comment by paulincai** [comment]
Hi @bratelefant the package is almost ready for a new release. Would be a good time to transition to another Cordova Push plugin which ideally ... should not be deprecated. I would be interested to work with you to test the new package version which now allows a separate DB connection for high volume push notifications. Would you like to take this to a private chat and work together? I would only need you to try your app with a package that I provide to you and we adjust for the best setup and finish everything with the right documentation.

**Comment by bratelefant** [comment]
Hey, I'm surely interested in testing; though time's a limiting factor right now. Which plugin are you thinking of, maybe [this one](https://github.com/havesource/cordova-plugin-push)? I'm currently using push on a private repo with ~1200 tokens, and pushing messages occurs "stackwise" (queues with ~300-500 push messages).

**Comment by paulincai** [comment]
Yes, that is the one. I see one of the forks of this package uses that package.

**Comment by bratelefant** [comment]
Is there a fork that you could possibly merge?

**Comment by paulincai** [comment]
Hi @bratelefant there are three ways I see to fix this.
1. You fork it and do your preferred changes.
2. You provide your updates in a fork and we merge into the main repo
3. Time is not a limiting factor anymore.

**Comment by bratelefant** [comment]
Hm. I'm actually thinking of prioritizing this one, since I think that push is essential to almost any meteor app and I'd really like to keep up with meteor core updates. 

I'd also really like to stick with Firebase, so maybe [this](https://github.com/chemerisuk/cordova-plugin-firebase-messaging) plugin could be promising?

**Comment by bratelefant** [comment]
I get iOS working, but Android devices do not receive background push notifications, although a device token is set and fcmConnection in `sendNotification` gives a result (log shows `Successfully sent message:...`). @paulincai  Any hint where to look at?

**Comment by paulincai** [comment]
Whatsapp please at +971 55 6036403 or skype at paulailincai (in United Arab Emirates).

**Comment by imongithubnow** [comment]
I'm also facing on this issue, there is news ?

**Comment by paulincai** [comment]
@imongithubnow are there any news?

**Comment by imongithubnow** [comment]
I tried some things but without lucky, I answered you on Meteor forums.
Seems clair the problem is in push component but so far I can't understand how resolve it.

**Comment by bratelefant** [comment]
My guess is that the problem is located in `phonegap-plugin-push`

**Comment by paulincai** [comment]
Hi @bratelefant where exactly?

**Comment by bratelefant** [comment]
Couldn't point my finger at, but moving to [cordova-plugin-push](https://github.com/havesource/cordova-plugin-push) at least eliminates this error and building is possible.

**Comment by HoptimizeME** [comment]
Hi,

is it still the case ? We were going to start using push notifications with our app based on Meteor 2.6.1. What do you guys do with your app ? You cannot upgrade to >2.5 because push notifications stopped working?

Regards

**Comment by bratelefant** [comment]
This can be fixed by a minor hack; there's also a Cordova plugin that automates this. Check this meteor forum thread for further details:  

https://forums.meteor.com/t/mobile-app-generation-cordova-issues-with-2-5-1-solved/57199/24

**Comment by rioadrian** [comment]
Hi All, 
first of all thanks to @paulincai for share this incredible plugin so push notification become easy and possible in meteor.
Just want to inform You that I have similar error when running android and it being solved by changing gradle 7.1.1 to 6.7.1 and trying running again with no problem. I hope it can help. And I also hoping for update so we can build on top of higer version gradle.

<img width="818" alt="Screen Shot 2022-05-20 at 17 02 28" src="https://user-images.githubusercontent.com/6780583/169505618-b0428cf6-4f5d-4ae5-8bdd-d394187ac71c.png">

Thanks & Regards

**Comment by rioadrian** [comment]
Hi All,
first of all thanks to @paulincai for share this incredible plugin so push notification become easy and possible in meteor.
Just want to inform You that I have similar error when running android and it being solved by changing gradle 7.1.1 to 6.7.1 and trying running again with no problem. I hope it can help. And I also hoping for update so we can build on top of higer version gradle.

<img width="670" alt="Screen Shot 2022-05-20 at 17 06 50" src="https://user-images.githubusercontent.com/6780583/169506110-3fe75469-2eb8-498b-b806-9e24642e0977.png">


Thanks & Regards
